components/link/create.js

```diff
@@ -36,6 +35,8 @@ const useStyles = makeStyles((theme) => ({
 }))
 
 export default function CreateLink(props) {
+  const baseUrl = window.location.origin
```

Comment:
The downside of this approach is that `window` will be undefined you go directly to `http://<host>:<port>/create` and you will have to navigate from `http://<host>:<port>` to reach it.

Perhaps you know a way to obtain `baseUrl` without relying on `window`. I wasn't able to get it working with using `react-router-dom` to extract `baseUrl`.

Reply:
I'm looking into this right now. I feel like there's a way to find what the current url is without using window.. Another option is to use a function that lets you wrap logic around this with default values.. etc.